src/public/modules/core/directives/on-click-outside.client.directive.js

```diff
@@ -0,0 +1,42 @@
+// Retrieved from https://codepen.io/robmazan/pen/gbeagj
+
+angular
+  .module('core')
+  .directive('onClickOutside', [
+    '$document',
+    '$parse',
+    '$timeout',
+    onClickOutside,
+  ])
+
+function onClickOutside($document, $parse, $timeout) {
+  return {
+    restrict: 'A',
+    compile: function (tElement, tAttrs) {
+      var fn = $parse(tAttrs.onClickOutside)
```

Comment:
excuse me sir are you from the past
```suggestion
      const fn = $parse(tAttrs.onClickOutside)
```